### PR TITLE
Add fancy yaml callback for Itxaka's pleasure


### PR DESCRIPTION
**Pull request by evrardjp**

### CODE REVIEW

File: script_library/pre-flight-checks.sh [hunk 1]
```diff
@@ -117,7 +117,11 @@ check_ansible_requirements (){
     # If ara is requested
     if [[ ${USE_ARA:-False} == "True" ]]; then
         check_python_requirement 'ara'
-        python3 -m ara.setup.env > ${SOCOK8S_WORKSPACE_BASEDIR}/${SOCOK8S_ENVNAME}-workspace/ara.rc
+        ARA_PATH=$(python3 -m ara.setup.path)
+        cat << EOF > ${SOCOK8S_WORKSPACE_BASEDIR}/${SOCOK8S_ENVNAME}-workspace/ara.rc
```
Comment (Itxaka) [inline script_library/pre-flight-checks.sh]: this sucks :(

Comment (evrardjp) [reply script_library/pre-flight-checks.sh]: I will propose a patch to ara I guess :)

Comment (evrardjp) [reply script_library/pre-flight-checks.sh]: https://review.opendev.org/#/c/679281/1

Comment (kantarun) [reply script_library/pre-flight-checks.sh]: Is not there need to generate env and output to ara.rc first ? And then append to file content new exports.

I mean this line (# 120) in existing code .
`python3 -m ara.setup.env > ${SOCOK8S_WORKSPACE_BASEDIR}/${SOCOK8S_ENVNAME}-workspace/ara.rc`

Comment (evrardjp) [reply script_library/pre-flight-checks.sh]: I will fix the line that's missing about the plugin location if that's what you meant.

Comment (kantarun) [reply script_library/pre-flight-checks.sh]: yes output of ara.setup.path though not sure if you still need that as you have already referred those paths in action and callback plugins env variables export

Comment (evrardjp) [reply script_library/pre-flight-checks.sh]: Still not sure what you mean here.

**Pull request closed without merging**